Terminal.Gui/Views/TextView.cs

```diff
@@ -2949,17 +3239,85 @@ void Adjust ()
 		public virtual void OnContentsChanged ()
 		{
 			ContentsChanged?.Invoke (this, new ContentsChangedEventArgs (CurrentRow, CurrentColumn));
+
+			ProcessInheritsPreviousColorScheme (CurrentRow, CurrentColumn);
+			ProcessAutocomplete ();
+		}
+
+		private void ProcessInheritsPreviousColorScheme (int row, int col)
```

Comment:
This method is used for persisting color scheme in each RuneCell on a file, if `InheritsPreviousColorScheme` property is enabled. When some text is inserted it will assume the color scheme of the previous rune cell color scheme. Sorry for the bad English.

Reply:
No need to apologize. It's all good!

For now, how about you add some  comments that say what you wrote above?

Reply:
If isn't enough please say, thanks. Fixed with the https://github.com/gui-cs/Terminal.Gui/pull/2682/commits/c974b57a9410c49eddfb88fd4a1138e864004c3a.